hexa/plugins/connector_dhis2/models.py

```diff
@@ -8,6 +8,7 @@
 from django.template.defaultfilters import pluralize
 from django.urls import reverse
 from django.utils import timezone
+from django.utils.text import slugify
```

Comment:
I would prefer if we use `python-slugify` instead (already installed within the project). Django's slugify does not cover some edge cases.